### PR TITLE
[PDI - 18988]: Fixed Username and Password exposed in cleartext on Server logs when adding or importing Hadoop Cluster

### PR DESCRIPTION
**Pull request by VootkoorSamhitha**
[PDI - 18988]: Fixed Username and Password exposed in cleartext on Server logs when adding or importing Hadoop Cluster

### CODE REVIEW

File: tomcat-logs/pom.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <groupId>pentaho</groupId>
+        <version>9.4.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>pentaho-tomcat-logs</artifactId>
+    <version>9.4.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```
Comment (peterrinehart) [inline tomcat-logs/pom.xml:17]: This is building on JDK11 now, and should use the parameter already defined here: https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L210

You shouldn't override this setting in lower projects unless it's essential; we're currently still targeting JDK8 for compatibility with certain big data drivers.

**Review by peterrinehart — CHANGES_REQUESTED** [review]

**Comment by peterrinehart** [comment]
We need to update the commit message to begin with [PDI-18988] and not Fixed - PDI-18988.  This ensures that our other tools will link this issue to the Jira ticket automatically.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 36m 59s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 28m
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Pull request merged**